libs/platform/src/lib/table/table.component.scss

```diff
@@ -415,6 +415,23 @@ $fd-table-border-var: var(--sapList_BorderWidth, 0.0625rem) solid var(--sapList_
     .fd-dnd-item {
         position: inherit;
     }
+
+    .fd-table__inner__icons {
```

Comment:
According to the [BEM](https://en.bem.info/methodology/quick-start/#guidelines-for-using-elements) we cannot have elements of element. 

> An element is always part of a block, not another element. This means that element names can't define a hierarchy such as block__elem1__elem2.

But basically these styles should be applied in Fundamental Styles, otherwise we need to refactor it later.